src/admin.js

```diff
@@ -458,6 +458,12 @@ export class StitchAdminClient extends StitchClient {
                   get: () => api._get(realmConfigUrl),
                   update: (data) => api._put(realmConfigUrl, { body: JSON.stringify(data) })
                 };
+              },
+              clientSchemas: () => {
+                const realmClientSchemasUrl = `${realmUrl}/client_schemas`;
+                return {
+                  get: language => api._get(`${realmClientSchemasUrl}/${language}`)
```

Comment:
This needs to set the body as well using `data` to handle namespaces. You'll need to figure out if we should do it like `_put` or `_post` though.

Reply:
Discussed offline, but we're gonna update the route to support namespaces in the query params as opposed to the body, since `node-fetch` doesn't allow bodies with GET requests

Reply:
We should update this to have (language, filter) =>